exe/kubernetes-deploy

```diff
@@ -15,18 +15,23 @@ require 'kubernetes-deploy'
 require 'optparse'
 
 skip_wait = false
+template_dir = nil
 ARGV.options do |opts|
   opts.on("-w", "--skip-wait") { skip_wait = true }
+  opts.on("-tDIR", "--template-dir=DIR") { |v| template_dir = v }
```

Comment:
I wonder if we should have the shorthands (`-w` and `-t`). We'd always spell them out in scripts, and this shouldn't really be run by hand—even then it's prone to error. Should we consider removing them?